Rakefile

```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp4
```

Comment:
and same here, master is for SP1 and not SP4, so please revert all these changes ( `git revert` is preffered way )